video/out/opengl/hwdec_cuda.c

```diff
@@ -334,24 +333,9 @@ static bool cuda_ext_vk_init(struct ra_hwdec_mapper *mapper,
     }
     mapper->tex[n] = ratex;
```

Comment:
A few lines above this, when you create the `pl_tex`, you still set `host_writable` even though you no longer need it (since you no longer call `pl_tex_upload`).